wysadzulice/assets/js/home.js

```diff
@@ -0,0 +1,65 @@
+import jquery from 'jquery';
+import { Model, View, Collection} from 'backbone';
+
+const BannerView = View.extend({
+
+  initialize: function(options) {
```

Comment:
use:
 * new, shorter version of method declaration:
```javascript
initialize() {
}
```
* use destructuring assigment to define parameters https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment